cl/phase1/stages/clstages.go

```diff
@@ -663,12 +664,19 @@ func ConsensusClStages(ctx context.Context,
 								})
 								// Attestations processing can take some time if they are not cached in properly.
 								go func() {
+									defer func() {
```

Comment:
The rationale is not that we expect it to panic, the rationale is that if an error happens we need to report it to the user but we want to avoid to crash the whole software for a recoverable error